src/semver/__init__.py

```diff
@@ -37,3 +37,5 @@
     __maintainer_email__,
     SEMVER_SPEC_VERSION,
 )
+
+__all__ = ["Version"]
```

Comment:
I fear, this is incomplete. It's not only `Version` we want to expose, but also the other imported functions, variables, and classes.

Could you add the other names as well?

Reply:
Including the deprecated ones?

Reply:
> Including the deprecated ones?

Yes. Some still use them so we can't omit them.